sosw/worker_assistant.py

```diff
@@ -96,19 +98,31 @@ def __call__(self, event):
                     raise Exception(f"Missing required parameter `{req_param}` in event for action `{action}`")
 
             func_kwargs = {k: event[k] for k in event if k in required_params}
+
+            if 'stats' in event:
+                func_kwargs.update({'stats': event['stats']})
+
+            if 'result' in event:
+                func_kwargs.update({'result': event['result']})
+
             return func(**func_kwargs)
         else:
             raise Exception(f"Action `{action}` is not supported")
 
 
-    def mark_task_as_completed(self, task_id: str):
+    def mark_task_as_completed(self, task_id: str, stats: dict, result=None):
```

Comment:
**Very bad!**
You do not support None value for `stats` argument here, while the `__call__` method supports it.
Of course you MUST support it, as long as otherwise you completely break support for all the older versions of `Worker` lambdas.